src/utils.rs

```diff
@@ -0,0 +1,129 @@
+#[cfg(not(target_arch = "wasm32"))]
+pub(crate) use native::*;
+#[cfg(target_arch = "wasm32")]
+pub use wasm::*;
+
+#[cfg(not(target_arch = "wasm32"))]
+mod native {
+    use std::sync::{
+        atomic::{AtomicBool, Ordering},
+        Arc,
+    };
+
+    use log::error;
+
+    pub(crate) struct Robot {
```

Comment:
There are three robots (k::Chain (variable name is robot), urdf_rs::Robot (This comes from the definition of URDF, it can not be changed), urdf_viz::utils::Robot).
So I want to rename this RobotModel or something to avoid confusion.

Reply:
renamed to RobotModel